app/views/people/_form.html.erb

```diff
@@ -52,7 +52,7 @@
         <%= f.input :address %>
       </div>
       <div class="col-12 col-md-6">
-        <%= f.input :birthday %>
+        <%= f.input :birthday, as: :string, input_html: { type: :date } %>
```

Comment:
`as: :date` breaks since it tries to convert it into a datetime for whatever reason.

by setting it to as a string and setting `input_html` fixes this issue. 

i don't know why it works, it just works sooo....